src/extension.ts

```diff
@@ -28,8 +28,17 @@ export namespace Openshift {
     export namespace Project {
         export const create = async function createProjectCmd(cli: cli.ICli, explorer: explorerFactory.OpenShiftExplorer, context: odoctl.OpenShiftObject) {
             const value  = await vscode.window.showInputBox({prompt: "Project name"});
-            await cli.execute(`odo project create ${value.trim()}`, {});
-            await explorer.refresh();
+            const characterRegex = /[a-z0-9]([-a-z0-9]*[a-z0-9])?/;
+            if(characterRegex.test(value)) {
+                if(value.length<=63) {
+                    await cli.execute(`odo project create ${value.trim()}`, {});
+                    await explorer.refresh();
+                } else {
+                    await vscode.window.showErrorMessage('Project name is to long');
+                }
+            } else {
+                await vscode.window.showErrorMessage('Project name should be alpha or alphe numerical');
```

Comment:
Why are 2 words alpha and alphe numerical used. It does not sound right.

Use this message: `Project name should be alphanumeric.`